### ISSUE TITLE
problem with 0 Sols/s

### ISSUE THREAD

**Issue by Doccos** [nicehash/nheqminer#119]
When i start a Bench with ./nheqminer_cpu -b

`Setting log level to 2
[16:33:28][0x00007f1aa1507740] Using SSE2: YES
[16:33:28][0x00007f1aa1507740] Using AVX: YES
[16:33:28][0x00007f1aa1507740] Using AVX2: YES
[16:33:28][0x00007f1aa1507740] Benchmarking CPU worker (CPU-XENONCAT-AVX2) 
[16:33:28][0x00007f1aa1507740] Benchmarking CPU worker (CPU-XENONCAT-AVX2) 
[16:33:28][0x00007f1aa1507740] Benchmarking CPU worker (CPU-XENONCAT-AVX2) 
[16:33:28][0x00007f1aa1507740] Benchmarking CPU worker (CPU-XENONCAT-AVX2) 
[16:33:28][0x00007f1aa1507740] Benchmarking CPU worker (CPU-XENONCAT-AVX2) 
[16:33:28][0x00007f1aa1507740] Benchmarking CPU worker (CPU-XENONCAT-AVX2) 
[16:33:28][0x00007f1aa1507740] Benchmarking CPU worker (CPU-XENONCAT-AVX2) 
[16:33:28][0x00007f1aa1507740] Benchmarking CPU worker (CPU-XENONCAT-AVX2) 
[16:33:28][0x00007f1aa1507740] Benchmark starting... this may take several minutes, please wait...
[16:33:53][0x00007f1aa1507740] Benchmark done!
[16:33:53][0x00007f1aa1507740] Total time : 24788 ms
[16:33:53][0x00007f1aa1507740] Total iterations: 200
[16:33:53][0x00007f1aa1507740] Total solutions found: 376
[16:33:53][0x00007f1aa1507740] Speed: 8.06842 I/s
[16:33:53][0x00007f1aa1507740] Speed: 15.1686 Sols/s
`

when i start mining with ./nheqminer_cpu -u "user.worker" -p pass  -l zec.suprnova.cc:2142 i get:

`[16:37:53][0x00007f673fbaf740] Using SSE2: YES
[16:37:53][0x00007f673fbaf740] Using AVX: YES
[16:37:53][0x00007f673fbaf740] Using AVX2: YES
[16:37:53][0x00007f673d8ae700] stratum | Starting miner
[16:37:53][0x00007f673cead700] miner#0 | Starting thread #0 (CPU-XENONCAT-AVX2) 
[16:37:53][0x00007f6737fff700] miner#1 | Starting thread #1 (CPU-XENONCAT-AVX2) 
[16:37:53][0x00007f67375fe700] miner#2 | Starting thread #2 (CPU-XENONCAT-AVX2) 
[16:37:53][0x00007f6736bfd700] miner#4 | Starting thread #4 (CPU-XENONCAT-AVX2) 
[16:37:53][0x00007f672ffff700] miner#3 | Starting thread #3 (CPU-XENONCAT-AVX2) 
[16:37:53][0x00007f67361fc700] miner#5 | Starting thread #5 (CPU-XENONCAT-AVX2) 
[16:37:53][0x00007f673d8ae700] stratum | Connecting to stratum server zec.suprnova.cc:2142
[16:37:53][0x00007f6734dfa700] miner#7 | Starting thread #7 (CPU-XENONCAT-AVX2) 
[16:37:53][0x00007f67357fb700] miner#6 | Starting thread #6 (CPU-XENONCAT-AVX2) 
[16:37:53][0x00007f673d8ae700] stratum | Connected!
[16:37:53][0x00007f673d8ae700] stratum | Subscribed to stratum server
[16:37:53][0x00007f673d8ae700] miner | Extranonce is 3601fa57000000000000000057fa0135
[16:38:03][0x00007f673fbaf740] Speed [300 sec]: 0 I/s, 0 Sols/s
[16:38:13][0x00007f673fbaf740] Speed [300 sec]: 0 I/s, 0 Sols/s
[16:38:23][0x00007f673fbaf740] Speed [300 sec]: 0 I/s, 0 Sols/s
`

**Comment by twatsnn** [comment]
add that parameter and try -e 2

**Comment by Doccos** [comment]
thanks for the answer

but it doesnt work :(

**Comment by chronosek** [comment]
i do not see in log that you get any job to do, your worker is not authorized or pool not working, it should be like:

[18:08:17][0x00001428] Using SSE2: YES
[18:08:17][0x00001428] Using AVX: YES
[18:08:17][0x00001428] Using AVX2: YES
[18:08:17][0x00001424] stratum | Starting miner
[18:08:17][0x00000634] miner#0 | Starting thread #0 (CPU-XENONCAT-AVX2)
[18:08:17][0x000016dc] miner#1 | Starting thread #1 (CPU-XENONCAT-AVX2)
[18:08:17][0x000017e8] miner#2 | Starting thread #2 (CPU-XENONCAT-AVX2)
[18:08:17][0x00000dac] miner#3 | Starting thread #3 (CPU-XENONCAT-AVX2)
[18:08:17][0x00000440] miner#4 | Starting thread #4 (CPU-XENONCAT-AVX2)
[18:08:17][0x00001424] stratum | Connecting to stratum server eu1-zcash.flypool.org:3333
[18:08:17][0x0000139c] miner#5 | Starting thread #5 (CPU-XENONCAT-AVX2)
[18:08:17][0x00001424] stratum | Connected!
[18:08:17][0x00001424] stratum | Subscribed to stratum server
[18:08:17][0x00001424] miner | Extranonce is 426f01e4ad
[18:08:17][0x00001424] stratum | Authorized worker t1bSsj8xWFJx6n2J1LDvjzuc8dEJ9r1DxaR.work1
[18:08:17][0x00001424] stratum | Target set to 0083126e978d4fdf3b645a1cac083126e978d4fdf3b645a1cac083126e978d4f
[18:08:18][0x00001424] stratum | Received new job #16855acfe5121508bf4f
[18:08:19][0x00001424] stratum | Received new job #0d111cb0237086951d00
[18:08:20][0x000016dc] stratum | Submitting share #4, nonce 010000000000000000000000000000000000000000000000000001
[18:08:27][0x00001428] Speed [300 sec]: 19.79608 I/s, 40.092 Sols/s

**Comment by Doccos** [comment]
hmmm i testet the account on a other machine and it works :/
but when i test it with flypool it works, dont know why it not works with suprnova

**Comment by thesios** [comment]
I am seeing the same issue with nanopool but only when running with cuda

**Comment by stilllife00** [comment]
Same issue with a cuda tromp build.

This is the command I run:

./nheqminer_cuda_tromp -u <my zcash address>.worker -cd 0 -l stratum.zcash.nicehash.com:3357

The entire OS freezes for a couple of seconds, then it works fine but with 0 Sols/s

The same happens if I with CPU instead of GPU.

Yesterday (dunno if related), running with Xorg disabled (plain console) it was working fine

**Comment by stilllife00** [comment]
Today after pulling the latest Linux updates , after starting the program I get this line

     [16:39:05][0x00007f6102ffd700] miner#7 | CUDA error 'the launch timed out and was terminated' in func 'solve' line 988

And the Sol/s are still at 0 . Any idea ? Shall I provide some logs ?

**Comment by S74nk0** [comment]
@stilllife00 try using CUDA SDK 7.5 this might fix  your the problem

**Comment by stilllife00** [comment]
Another info: running with -e 0 (which disables AVX2) I get it to run at 11 Sol/s
In parallel another instance just runs on CPU at 15Sol/s
Not sure it's the best thing to do though. I will try also the older SDK

Given a GeForce GTX970m what would be a reasonable Sol/s ?

**Comment by thesios** [comment]
I wish i could get my gpu working 

have anyone tried a aws ec2 p2 type instance ?

**Comment by ronakthamor** [comment]
with p2.16xlarge I am just getting 130 sol/sec . ZENcash.. Am I doing something wrong?

Command:

    ./nheqminer_cuda_tromp -l us.zenmine.pro:9009 -u ZEN_ADDR.aws -p x -t 64 -cd 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15

**Comment by thesios** [comment]
I am not sure if that is good or bad 

What image did u use?  or would be willing to save the image and save it ?

**Comment by ronakthamor** [comment]
Ubuntu 16.04. I have configured the server manually. This is what I followed:

https://steemit.com/zencash/@rnkhouse/zencash-cloud-mining-ubuntu-16-04

**Comment by thesios** [comment]
i seen that note , maybe I will try it , what base ami did u use inorder to have the correct cuda drivers ?

**Comment by thesios** [comment]
thats are a bunch of other miners, u may want to try some of the other ones and see which one will pay you better

**Comment by ronakthamor** [comment]
Yeah.. Let me know how it works for you. You can also comment on steemit post.. ubuntu 16.04 is the base ami.

**Comment by apurveanand** [comment]
<img width="588" alt="screen shot 2018-03-07 at 4 24 57 am" src="https://user-images.githubusercontent.com/32944429/37084080-8457e726-21bf-11e8-887d-5fc094972afb.png">

My nheqminer is giving this log. Its ignoring many new jobs mentioning non clean job. How to fix this?

**Comment by cen94ok** [comment]
![image](https://user-images.githubusercontent.com/30008749/55737394-c7957600-5a2d-11e9-91f3-6849c27561dd.png)

How to fix it? So strange :\
I have signed up on suprnova, but it's so
